build_a_container.rst

```diff
@@ -149,26 +147,25 @@ You can do so with the following command.
 
 .. code::
 
-   $ sudo {command} build lolcow.sif lolcow.def
-
-The command requires ``sudo`` just as installing software on your local
-machine requires root privileges.
+   $ {command} build lolcow.sif lolcow.def
 
 .. note::
 
    Beware that it is possible to build an image on a host and have the
    image not work on a different host. This could be because of the
    default compressor supported by the host. For example, when building
    an image on a host in which the default compressor is ``xz`` and then
-   trying to run that image on a CentOS 6 node, where the only
+   trying to run that image on a node where the only
    compressor available is ``gzip``.
 
 *******************************
  Building encrypted containers
 *******************************
 
 With {Project} it is possible to build and run
-encrypted containers. The containers are decrypted at runtime entirely
+encrypted containers. 
+When running in setuid-mode,
```

Comment:
We should make it clearer that setuid-mode is the only way to run an encrypted container.